### PR TITLE
Allow to reject specific languages on receiving posts via the relay

### PR DESCRIPTION
**Pull request by annando**
When receiving posts via the relay, we receive posts in many different languages. We now can reject specific languages. Since many European languages are really similar, this will not work great on removing a specific language from that group of languages. But it helps with - for example - rejecting posts in Japanese when you don't have got a Japanese audience on your system.

### CODE REVIEW

File: static/defaults.config.php [hunk 1]
```diff
@@ -525,6 +525,14 @@
 		// The authentication password for the redis database
 		'redis_password' => null,
 
+		// relay_deny_languages (Array)
+		// Array of languages that are rejected.
+		'relay_deny_languages' => [],
```
Comment (MrPetovan) [inline static/defaults.config.php:530]: What's the format for the language array? [ISO 693-1](https://en.wikipedia.org/wiki/ISO_639-1) or [IETF BCP 47](https://en.wikipedia.org/wiki/IETF_language_tag)?

Comment (annando) [reply static/defaults.config.php:530]: Well, it's the same two digit format that we use elsewhere.

Comment (MrPetovan) [reply static/defaults.config.php:530]: Please mention the expected format (2-letter language code) here as well.

Comment (annando) [reply static/defaults.config.php:530]: done

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**